### PR TITLE
[13.0][MIG] account_bank_statement_import_qif: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Superseed https://github.com/OCA/bank-statement-import/pull/342

Migration to 13.0

Please @pedrobaeza can you review it?

@Tecnativa TT26069

### CODE REVIEW

File: account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py
Comment (pedrobaeza) [inline account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py]: Please don't create a new field. Use existing method `_check_qif`. Anyway, is this really needed?

Comment (victoralmau) [reply account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py]: This is necessary so that other modules do not execute that code that is only needed in this module. 

Perhaps another possible approach would be to inject it in context here https://github.com/odoo/odoo/blob/13.0/addons/account_bank_statement_import/account_bank_statement_import.py#L50 but I didn't want to do that in order not to repeat code. 
What do you think about this?

Comment (pedrobaeza) [reply account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py]: OK, but it should be a method, not a field.

File: account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py
Comment (pedrobaeza) [inline account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py]: Why 2 methods? Just use `_check_qif`

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I think we can finally merge this one without waiting for a second review, after being all this time queued, and being tested:

/ocabot merge nobump

**Pull request merged**
